### PR TITLE
add section to explain how to retrieve admin pw

### PR DESCRIPTION
**Pull request by itdove**

### CODE REVIEW

File: stories/topics/proc-aws-application-deploy.adoc [hunk 1]
```diff
@@ -29,3 +29,13 @@ In the *Capabilities section*, check to acknowledge that CloudFormation may crea
 
 The application begins provisioning.  
 It can take some time for the infrastructure and application to fully provision.
+
+To retrieve the admin password:
+
```
Comment (ianf77) [inline stories/topics/proc-aws-application-deploy.adoc:34]: Add .Procedure 
immediately before the procedure.

File: stories/topics/proc-aws-application-deploy.adoc [hunk 2]
```diff
@@ -29,3 +29,13 @@ In the *Capabilities section*, check to acknowledge that CloudFormation may crea
...
+. Select the main menu from the top left.
+. Select *Security*. If you do not see *Security*, then select *View All Products*.
```
Comment (ianf77) [inline stories/topics/proc-aws-application-deploy.adoc:36]: delete 'then'

File: stories/topics/proc-aws-application-deploy.adoc [hunk 3]
```diff
@@ -29,3 +29,13 @@ In the *Capabilities section*, check to acknowledge that CloudFormation may crea
...
+. Select *Secret Manager*.
+. Filter with the name of the deployment as the secret name format is <DeploymentName>-aap-admin
+. Click on the secret name of the deployment
+. Click on the 3 vertical dots on the line of the deployment.
+. Select *View secret value*, the admin password will be displayed.
```
Comment (ianf77) [inline stories/topics/proc-aws-application-deploy.adoc:41]: administration

**Comment by itdove** [comment]
@ianf77 Sorry, I made the change in the aws file instead of gcp. I corrected it and now is in the gcp one.

File: stories/topics/proc-aap-gcp-application-deploy.adoc [hunk 4]
```diff
@@ -53,3 +53,15 @@ This deployment has resources from the Runtime Configurator service, which is in
...
+To retrieve the admin password:
+
+.Procedure
+. Select the main menu from the top left.
+. Select *Security*. If you do not see *Security*, select *View All Products*.
+. Select *Secret Manager*.
+. Filter with the name of the deployment as the secret name format is <DeploymentName>-aap-admin
+. Click on the secret name of the deployment
+. Click the image::more_actions.png[More,10,25] on the line of the deployment.
+. Select *View secret value*, the administration password will be displayed.
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-deploy.adoc:66]: the administration password is displayed.

File: stories/topics/proc-aap-gcp-application-uninstall.adoc [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+If a new service account was created for the deployment and if it is not use anywhere else, it could be deleted
+====
+
+.Procedure
+. Select the main menu from the top left.
+. Select *IAM & Admin*. If you do not see *IAM & Admin*, select *View All Products*.
+. Select *Service Accounts*.
+. Filter with the name of the service account.
+. Click the image::more_actions.png[More,10,25] on the line of the service account.
+. Select *Delete*, the administration password will be displayed.
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-uninstall.adoc:49]: is displayed

File: stories/topics/proc-aap-gcp-application-uninstall.adoc [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+====
+If a Load Balancer was added for the deployment, it needs to be deleted before uninstalling the deployment.
+====
+
+.Procedure
+. Select the main menu from the top left.
+. Select *Deployment Manager*. If you do not see *Deployment Manager*, select *View All Products*.
+. Select *Deployment Manager*.
+. Tick the checkbox in front of the deployment to be deleted.
+. Click on *Delete* in the top bar, this action can not be reversed.
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-uninstall.adoc:20]: Click btn:[Delete]

File: stories/topics/proc-aap-gcp-application-uninstall.adoc [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+.Procedure
+. Select the main menu from the top left.
+. Select *Security*. If you do not see *Security*, select *View All Products*.
+. Select *Secret Manager*.
+. Filter with the name of the deployment as the secret name format is <DeploymentName>-aap-admin
+. Click the image::more_actions.png[More,10,25] on the line of the deployment.
+. Select *Delete*, the administration password will be displayed. 
+. Type the name of the secret.
+. Click *Delete Secret*
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-uninstall.adoc:34]: Click btn:[Delete Secret].

File: stories/topics/proc-aap-gcp-application-uninstall.adoc [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+====
+
+.Procedure
+. Select the main menu from the top left.
+. Select *IAM & Admin*. If you do not see *IAM & Admin*, select *View All Products*.
+. Select *Service Accounts*.
+. Filter with the name of the service account.
+. Click the image::more_actions.png[More,10,25] on the line of the service account.
+. Select *Delete*, the administration password will be displayed. 
+. Click *Delete*
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-uninstall.adoc:50]: . Click btn:[Delete].

File: stories/topics/proc-aap-gcp-application-uninstall.adoc (hunk 7 above)
Comment (itdove) [reply stories/topics/proc-aap-gcp-application-uninstall.adoc:34]: This is not working, when I generate the doc I see `Click btn:[Delete Secret]`

Comment (ianf77) [reply stories/topics/proc-aap-gcp-application-uninstall.adoc:34]: Is that the only btn:[] that doesn't resolve?

Comment (itdove) [reply stories/topics/proc-aap-gcp-application-uninstall.adoc:34]: No also some `image:...` are not working

File: stories/topics/proc-aap-gcp-application-deploy.adoc [hunk 9]
```diff
@@ -53,3 +53,15 @@ This deployment has resources from the Runtime Configurator service, which is in
...
+
+To retrieve the admin password:
+
+.Procedure
+. Select the main menu from the top left.
+. Select *Security*. If you do not see *Security*, select *View All Products*.
+. Select *Secret Manager*.
+. Filter with the name of the deployment as the secret name format is <DeploymentName>-aap-admin
+. Click on the secret name of the deployment
+. Click the image::more_actions.png[More,10,25] on the line of the deployment.
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-deploy.adoc:65]: It should be image:more_actions.png

File: stories/topics/proc-aap-gcp-application-uninstall.adoc [hunk 10]
```diff
@@ -0,0 +1,50 @@
+[id="proc-aap-gcp-application-uninstall_{context}"]
+
+= Uninstall Application
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-uninstall.adoc:3]: Change to: Uninstalling the application

File: stories/topics/proc-aap-gcp-application-uninstall.adoc [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+.Procedure
+. Select the main menu from the top left.
+. Select *Deployment Manager*. If you do not see *Deployment Manager*, select *View All Products*.
+. Select *Deployment Manager*.
+. Tick the checkbox in front of the deployment to be deleted.
+. Click on *Delete* in the top bar, this action cannot be reversed.
+
+The deployment is being deleted.
+
+To delete the administration secret.
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-uninstall.adoc:24]: Change to == Deleting the administration secret

File: stories/topics/proc-aap-gcp-application-uninstall.adoc [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+. Select the main menu from the top left.
+. Select *Security*. If you do not see *Security*, select *View All Products*.
+. Select *Secret Manager*.
+. Filter with the name of the deployment as the secret name format is <DeploymentName>-aap-admin
+. Click the image::more_actions.png[More,10,25] on the line of the deployment.
+. Select *Delete*, the administration password will be displayed. 
+. Type the name of the secret.
+. Click *Delete Secret*
+
+To delete the service account.
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-uninstall.adoc:36]: Change to: == Deleting the service account

File: stories/assembly-aap-gcp-uninstall.adoc [hunk 13]
```diff
@@ -0,0 +1,8 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-aap-gcp-uninstall"]
+= Uninstalling Red Hat {AAPonGCP}
+
+:context: aap-gcp-uninstall
+
```
[the remark this answers is not in the document]
Comment (itdove) [reply stories/assembly-aap-gcp-uninstall.adoc:7]: then we have the same issue with assembly-aap-gcp-install.adoc

File: stories/topics/proc-aap-gcp-application-uninstall.adoc [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+.Procedure
+. Select the main menu from the top left.
+. Select *Deployment Manager*. If you do not see *Deployment Manager*, select *View All Products*.
+. Select *Deployment Manager*.
+. Tick the checkbox in front of the deployment to be deleted.
+. Click on *Delete* in the top bar, this action cannot be reversed.
+
+The deployment is being deleted.
+
+Deleting the administration secret.
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-uninstall.adoc:21]: == Deleting the administration secret 
(making it a sub heading)

File: stories/topics/proc-aap-gcp-application-uninstall.adoc [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+. Select the main menu from the top left.
+. Select *Security*. If you do not see *Security*, select *View All Products*.
+. Select *Secret Manager*.
+. Filter with the name of the deployment as the secret name format is <DeploymentName>-aap-admin
+. Click the image::more_actions.png[More,10,25] on the line of the deployment.
+. Select *Delete*, the administration password will be displayed. 
+. Type the name of the secret.
+. Click btn:[Delete Secret].
+
+Deleting the service account.
```
Comment (ianf77) [inline stories/topics/proc-aap-gcp-application-uninstall.adoc:33]: == Deleting the service account

(sub heading)

**Review by ianf77 — APPROVED** [review]
LGTM

Shall I merge?

**Comment by itdove** [comment]
@ianf77 yes as it seems I don't have the permission to do so and that info is vaild for the current release

**Pull request merged**